.github/workflows/validate-releasenotes.yml

```
@@ -0,0 +1,28 @@
+name: Validate release notes
+on:
+  push:
+    branches:
+      - 'main'
+      - 'release/**'
+  pull_request:
+    types: [opened, reopened, labeled, unlabeled, synchronize]
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10.9'
+      - name: Install pip packages
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Validate releasenotes
+        if: github.event_name == 'pull_request' && !contains(github.event.pull_request.labels.*.name, 'no-reno')
+        env:
+          BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+        run: ./.github/renos_updated.sh
```

```suggestion
      - name: Lint releasenotes
        run: reno lint
      - name: Check releasenotes updated
        if: github.event_name == 'pull_request' && !contains(github.event.pull_request.labels.*.name, 'no-reno')
        run: ./.github/renos_updated.sh
```
It seems a bit clearer to me to always lint the release notes and then only check if something was added in pull requests without the `no-reno` label. I also couldn't find the `BRANCH_NAME` used anywhere.

---

Completed in 737697a